packages/agreement-process/src/services/agreementCreationProcessor.ts

```diff
@@ -28,9 +28,6 @@ export async function createAgreementLogic(
   tenantQuery: TenantQuery,
   correlationId: string
 ): Promise<CreateEvent<AgreementEvent>> {
-  logger.info(
-    `Creating agreement for EService ${agreement.eserviceId} and Descriptor ${agreement.descriptorId}`
-  );
```

Comment:
Unrelated - this log is duplicated, it is performed in the same way also in `agreementService.ts`

Reply:
Done with [this commit](https://github.com/pagopa/interop-be-monorepo/pull/392/commits/f34b7ba3bfa9c80e8dc54e0c6e25f3e833d4b839)   in another [PR](https://github.com/pagopa/interop-be-monorepo/pull/392)